ocs_ci/utility/vsphere_nodes.py

```diff
@@ -85,6 +85,31 @@ def restart_chrony(self):
             _rc = True
         return _rc
 
+    def set_host_name(self, host_name):
+        """
+        Sets the host name
+
+        Args:
+            host_name (str): Name to set as host name
+
+        Returns:
+            tuple: tuple which contains command return code, output and error
```

Comment:
do we really need to return something here? Will we use those data somewhere? Or the raise an exception is enough when it is not successful?

Reply:
By default exec_cmd returns the ret code, output and error. It upto the caller to use it or not. here we are not using the data.
We need to set the host name after cloning the template and reboot the node to make it effect.